### ISSUE TITLE
Training details question

### ISSUE THREAD

**Issue by YNX940214** [twhui/MSG-Net#6]
Hi , it is generous of you to share the preprocess code.
I am trying to implement the paper with pytorch but I have some questions about the training procedure.

~~1 How much time does it take to train the model?  The paper says "We trained each MS-Net and MSG-Net for 5e+5 iterations."  It seems  will take such a long time on my device (GTX 1080 Ti) ....... For i implemented Deep Joint Filter (another paper with similar architecture with SRCNN) for 2000 epoch and it takes about 2 days then 500000 may take nearyly one year.~~

~~2 "we used stepwise decrease steps with learning rate multiplier γ = 0:8)" .  Does that mean 0 to 9999 epoch is lr, then 10000 to 19999 use lr*0.8 and then 20000 to 29999 use lr*0.64?~~

1,2  What is the batchSize for trainning 5e5 iterations? 

3 "We the following pre-training and fine-tuning scheme for MSG-Net: (1) we pre-trained the Y - and D- branches for a 2× MSG-Net separately, (2) we transfered the first two layers of them (D-branch: fconv1, deconv2g and Y -branch: fconv1Y, conv2Yg) to a plain 2× MSG-Net and then fine-tuned it. For training
MSG-Net with other upsampling factors (2m; m > 1), we transfered all the layers except the last four layers in the D-branch from the network trained with upsampling factor 2m−1 to a plain network and then fine-tuned it."
Can you explain the details a little more? My understanding is that : train the 2x model first, copy the weights to the 4x model then train 4x model, then copy the weights to 8x model..... Am i right?

4"To synthesize LR depth samples {Dl} , we first filtered each full-resolution sub-image by a 2D Gaussian kernel and then downsampled it by the given scaling factor."
That is not shown in the released code?
